src/test/kotlin/autolotto/LottoTest.kt

```diff
@@ -0,0 +1,122 @@
+package autolotto
+
+import autolotto.vo.AutoLotto
+import autolotto.vo.Lotto
+import autolotto.vo.WinningLotto
+import autolotto.winningpoint.WinningRank
+import io.kotest.matchers.ints.beInRange
+import io.kotest.matchers.shouldBe
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class LottoTest {
+    @Test
+    fun `로또 1장은 6개의 숫자다`() {
+        // Given
+        val lotto = Lotto()
+
+        // When
+        val numbers = lotto.numbers
+
+        // Then
+        numbers.size shouldBe 6
+    }
+
+    @Test
+    fun `로또 1장의 숫자는 1~45의 숫자다`() {
+        // Given
+        val lotto = Lotto()
+
+        // When
+        val numbers = lotto.numbers
+
+        // Then
+        numbers.forEach {
+            it shouldBe beInRange(IntRange(1, 45))
```

Comment:
```suggestion
            it shouldBeInRange IntRange(1, 45)
```

p3; 작성해 주신 걸 shouldBeInRange를 통해 줄여볼 수 있을 거 같아요.

Reply:
요거 제가 트라이 해봤는데 왜때문인지 버전을 올려도 적용이 안되더라구요 ;;; ㅠㅠ 분명 있는데 IDE에서 안써지는 요상한 상황..
<img width="680" alt="스크린샷 2023-11-09 오후 8 18 04" src="https://github.com/next-step/kotlin-lotto/assets/44457591/50677ce6-1a48-4c8b-a05f-f751a94cf616">